eiffel-vocabulary/EiffelConfigurationOrderedEvent.md

```diff
@@ -0,0 +1,19 @@
+
+# EiffelConfigurationOrderedEvent (ArtS)
+
+The EiffeliConfigurationOrderedEvent declares that a configuration has been Ordered.
```

Comment:
I have add more clarity to the description

__Description:__ The EiffeliConfigurationOrderedEvent declares that a new configuration has been ordered and is awaiting implementation. This event is followed by EiffelConfigurationAppliedEvent.

"Ordered" is now "ordered"